lavalink/WebSocket.py

```diff
@@ -128,6 +134,9 @@ def connected(self):
                     await self._lavalink.dispatch_event(event)
             elif op == 'playerUpdate':
                 await self._lavalink.update_state(data)
+            elif op == 'stats':
+                self._lavalink.stats._update(data)
+                await self._lavalink.dispatch_event(RawStatsUpdateEvent(data))

```

Comment:
Perhaps the `stats` could be dispatched with the event in place of `data` so that users can do `stats.cpu.system_load` etc rather than `data['cpu']['systemLoad']`

Reply:
Shall we rename the `RawStatsUpdateEvent` to `StatsUpdateEvent` then?

Reply:
Yes